module/Motion/BonusBlockCarrier.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file BonusBlockCarrier.cpp
+ * @brief ボーナスブロックをベースエリアまで運搬する
+ * @author sugaken0528
+ */
+
+#include "BonusBlockCarrier.h"
+
+void BonusBlockCarrier::carryBonusBlock()
+{
+  Rotation rotation;
+  StraightRunner straightRunner;
+  LineTracer lineTracer(true);
+  Controller controller;
+
+  int targetBrightness = (93 - 3) / 2;  // 目標輝度
+
+  // 交点内を直進
+  straightRunner.run(5, 50);
+
+  // 左に90度ピボットターン
+  rotation.turnForwardLeftPivot(89, 70);
+
+  // 赤を認識するまでライントレース
+  lineTracer.runToColor(COLOR::RED, targetBrightness, 40, PidGain(0.1, 0.08, 0.08));
+
+  // 左に90度ピボットターン
+  rotation.turnForwardLeftPivot(91, 87);
+
+  // 直進を安定させるために1秒待機
+  controller.sleep(1000000);
+
+  // 緑を認識するまで直進(指定した距離まで直進で代用)
```

Comment:
`TODO: ~`って形でコメントしとけば，後で検索もできるしいいかも
```suggestion
  // 緑を認識するまで直進
  // TODO: 指定した距離まで直進で代用してるので，指定した色まで直進に変更する
```